extensions/amp-access/0.1/amp-login-done-dialog.js

```diff
@@ -203,9 +203,23 @@ export class LoginDoneDialog {
       // Ignore.
     }
 
+    // Keep trying to close the window.
+    // Sometimes `window.close()` is ignored by the browser for a stretch of time.
+    // For instance, iOS ignores the method when there is a
+    // "Save Password" or "Update Password" prompt open.
+    // https://github.com/ampproject/amphtml/issues/11369
+    const windowCloseInterval = this.win.setInterval(() => {
+      try {
+        this.win.close();
+      } catch (e) {
+        // Ignore.
+      }
+    }, 100);
+
     // Give the opener a chance to close the dialog, if not, show the
     // close button.
     this.win.setTimeout(() => {
+      clearInterval(windowCloseInterval);
       this.postbackError_(new Error('Failed to close the dialog'));
     }, 3000);
```

Comment:
Is three seconds long enough to catch most users handling the password save popup? Would there be any harm in making it longer (5? 10?). @dvoytenko WDYT is a good value here?

Reply:
That's a good point. Honestly, I think it'd be safe to remove this `clearInterval` and have it keep trying indefinitely

Reply:
Let's go for 60 seconds.  i don't like loops that run forever. :-)

Reply:
Done!